frontend/src/components/GrantAccess.vue

```diff
@@ -36,60 +54,59 @@ async function grantAccess() {
   const grantAccessRequest = toRequest(formData.value)
   try {
     await apiService.grantUserRole(grantAccessRequest)
-    toast.success(`User "${grantAccessRequest.user_name}"" is granted with "${formData.value.role.role_name}" access.`)
-    formData.value = JSON.parse(JSON.stringify(defaultFormData)) // clone default input data.
+    toast.success(`User "${grantAccessRequest.user_name}" is granted with "${formData.value.role.role_name}" access.`)
+    formData.value = {...defaultFormData}; // clone default input data.
```

Comment:
Same as comment on line 21. Thanks Nick!

Reply:
I have switched back to the previous implementation.